src/cosmosdb-preview/azext_cosmosdb_preview/tests/latest/test_cosmosdb-cassandrami_scenario.py

```diff
@@ -20,11 +20,13 @@ def test_managed_cassandra_cluster_without_datacenters(self, resource_group):
 
         self.kwargs.update({
             'c': self.create_random_name(prefix='cli', length=10),
-            'subnet_id': self.create_subnet(resource_group)
+            'subnet_id': self.create_subnet(resource_group),
+            'seed_nodes': '127.0.0.1,127.0.0.2',
```

Comment:
The real way to pass in `nargs=+` parameter value is to give all values seperated by white space ` `.
So `seed_nodes` here could be `127.0.0.1 127.0.0.2`.

Reply:
fixed.

Reply:
Do you fix it locally? Don't forget to push code~

Reply:
Haha yes. Code incoming.